pydicom/uid.py

```diff
@@ -239,55 +248,56 @@ def is_valid(self) -> bool:
         return False
 
 
-# Pre-defined Transfer Syntax UIDs (for convenience)
-ImplicitVRLittleEndian = UID('1.2.840.10008.1.2')
-"""1.2.840.10008.1.2"""
-ExplicitVRLittleEndian = UID('1.2.840.10008.1.2.1')
-"""1.2.840.10008.1.2.1"""
-DeflatedExplicitVRLittleEndian = UID('1.2.840.10008.1.2.1.99')
-"""1.2.840.10008.1.2.1.99"""
-ExplicitVRBigEndian = UID('1.2.840.10008.1.2.2')
-"""1.2.840.10008.1.2.2"""
-JPEGBaseline8Bit = UID('1.2.840.10008.1.2.4.50')
-"""1.2.840.10008.1.2.4.50"""
-JPEGExtended12Bit = UID('1.2.840.10008.1.2.4.51')
-"""1.2.840.10008.1.2.4.51"""
-JPEGLosslessP14 = UID('1.2.840.10008.1.2.4.57')  # needs to be updated
-"""1.2.840.10008.1.2.4.57"""
-JPEGLosslessSV1 = UID('1.2.840.10008.1.2.4.70')  # Old JPEGLossless
-"""1.2.840.10008.1.2.4.70"""
-JPEGLSLossless = UID('1.2.840.10008.1.2.4.80')
-"""1.2.840.10008.1.2.4.80"""
-JPEGLSNearLossless = UID('1.2.840.10008.1.2.4.81')
-"""1.2.840.10008.1.2.4.81"""
-JPEG2000Lossless = UID('1.2.840.10008.1.2.4.90')
-"""1.2.840.10008.1.2.4.90"""
-JPEG2000 = UID('1.2.840.10008.1.2.4.91')
-"""1.2.840.10008.1.2.4.91"""
-JPEG2000MCLossless = UID('1.2.840.10008.1.2.4.92')
-"""1.2.840.10008.1.2.4.92"""
-JPEG2000MC = UID('1.2.840.10008.1.2.4.93')
-"""1.2.840.10008.1.2.4.93"""
-MPEG2MPML = UID('1.2.840.10008.1.2.4.100')
-"""1.2.840.10008.1.2.4.100"""
-MPEG2MPHL = UID('1.2.840.10008.1.2.4.101')
-"""1.2.840.10008.1.2.4.101"""
-MPEG4HP41 = UID('1.2.840.10008.1.2.4.102')
-"""1.2.840.10008.1.2.4.102"""
-MPEG4HP41BD = UID('1.2.840.10008.1.2.4.103')
-"""1.2.840.10008.1.2.4.103"""
-MPEG4HP422D = UID('1.2.840.10008.1.2.4.104')
-"""1.2.840.10008.1.2.4.104"""
-MPEG4HP423D = UID('1.2.840.10008.1.2.4.105')
-"""1.2.840.10008.1.2.4.105"""
-MPEG4HP42STEREO = UID('1.2.840.10008.1.2.4.106')
-"""1.2.840.10008.1.2.4.106"""
-HEVCMP51 = UID('1.2.840.10008.1.2.4.107')
-"""1.2.840.10008.1.2.4.107"""
-HEVCM10P51 = UID('1.2.840.10008.1.2.4.108')
-"""1.2.840.10008.1.2.4.108"""
-RLELossless = UID('1.2.840.10008.1.2.5')
-"""1.2.840.10008.1.2.5"""
+with disable_value_validation():
```

Comment:
Is this necessary?

Reply:
It's not, it's more of a debugging aid for the validation itself - to avoid running into the validation of each predefined UID in each test. It's also a usage example... Performacne-wise it shouldn't matter, though.